ui/app/templates/components/job-editor/edit.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="boxed-section">
+    <div class="boxed-section-head">
+    Job Definition
+    {{#if @data.cancelable}}
+        <button
+            class="button is-light is-compact pull-right"
+            onclick={{fn @fns.onCancel}}
```

Comment:
this'll throw you a requires at least 1 argument error; try this:

```suggestion
            onclick={{@fns.onCancel}}
```

Reply:
The tests would capture that, no? Per the [RFC](https://github.com/emberjs/rfcs/blob/master/text/0470-fn-helper.md) the `fn` helper helps with:

- binding the `this` context
- applying a list of arguments
- partial application to prevent prop drilling

None of these apply and we have test coverage for this. I manually tested this and not seeing this error either.

Can you reproduce?

Reply:
The first line of that RFC includes "to allow clear argument passing for functions in templates"; fn is for passing args and you're not passing any here. See also the current lint error on this PR.

Reply:
Totally missed that. I read this as vice-versa. Like you were advocating for that change. So disregard my comments and good catch because I made a mistake here.